modules/extension/wmts/src/main/java/org/geotools/ows/wmts/client/WMTSTileFactory.java

```diff
@@ -69,13 +69,15 @@ public Tile findTileAtCoordinate(
     }
 
     /** Find the first valid Upper Left tile for the current layer. */
+    @Deprecated
```

Comment:
So these public methods are gone, and there is no replacement? I don't think they are widely used, but still... would be nice if there was a way to get the same functionality after the refactor. If there is, the javadoc should say what it is (and the upgrade rst file too).

Reply:
Sorry, I was a little quick there. I will take the deprecation with its own issue at a later stage.

Reply:
@aaime You've changed the status of the ticket in JIRA to Solved, but the PR isn't committed.

Reply:
Whoops... easily fixed.